.github/workflows/ci_backend_go.yaml

```
@@ -0,0 +1,55 @@
+name: CI GO Backend
+on:
+  push:
+    branches: [main]
+    paths:
+      - ".github/workflows/ci_backend_go.yaml"
+      - "backend-go/**"
+    paths-ignore:
```

actionlint (1.7.4)

both "paths" and "paths-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)